src/modules/material-requests/queries/delete-material-request.graphql

```diff
@@ -0,0 +1,7 @@
+mutation deleteMaterialRequest($materialRequestId: uuid, $userProfileId: uuid) {
+	delete_app_material_requests(
+		where: { id: { _eq: $materialRequestId }, profile_id: { _eq: $userProfileId } }
```

Comment:
maybe also check is_pending _eq true
since we do not want users to be able to delete material requests that were already sent to a maintainer